pyntcloud/core_class.py

```diff
@@ -629,6 +629,7 @@ def plot(self,
              background="black",
              mesh=False,
              use_as_color=["red", "green", "blue"],
+             init_point_size=0.0,
```

Comment:
why "init" instead of "initial"?? Me english is not very good but I found more intuitive "initial_point_size"

Reply:
The main reason was to just keep the length of the argument name as short as possible. I thought it should be understandable because I wrote "the initial size of each point" in the docstring. I will change it to be explicit, as you suggested  :smile: